kotlin/build.gradle

```diff
@@ -94,10 +94,11 @@ buildscript {
           'androidSupport': "com.android.support:support-annotations:26.0.2",
       ],
       'test': [
-          'truth': "com.google.truth:truth:${versions.truth}",
+          'core_testing': "androidx.arch.core:core-testing:2.0.0-rc01",
```

Comment:
This key should mention androidx

Reply:
There are a few others that should as well. I'm going to put up another PR regrouping them to a common `androidx` parent.

Reply:
https://github.com/square/workflow/pull/458